src/extension.ts

```diff
@@ -664,7 +669,7 @@ function launchMetals(
                 command: ServerCommands.CopyWorksheetOutput,
                 arguments: [uri.toString()],
               })
-              .then((result) => {
+              .then((result: any) => {
```

Comment:
`any` is dangerous because it basically turns off the typechecker.

Why do we need a cast here? What's the type of `result` if we don't cast it?

Reply:
Wait, did I change that? 
If I did, it was due to trying some bug fixing and forgetting to remove it, so I believe it can go back to no `any` 😅